lib-find-meal/src/main/java/com/kinandcarta/lib/find/meal/FindMealActivity.kt

```diff
@@ -47,7 +45,7 @@ class FindMealActivity : AppCompatActivity() {
 
     override fun onDestroy() {
         super.onDestroy()
-        mainScope.cancel()
+        mainScope.cancel() // TODO: We'd get this for free in a ViewModel?
```

Comment:
I'd really recommend using view models, because on Android they get lifecycle support for free

Reply:
@dsmith42 @roger-tan are you both working on this? :)